recipes/grpc/all/patches/v1.46.x/0001-fix-cmake.patch

```diff
@@ -0,0 +1,10 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -5,6 +5,3 @@ include(conanbuildinfo.cmake)
+ conan_basic_setup(KEEP_RPATHS)
+
+ add_subdirectory("source_subfolder")
+-
+-# TODO: move to a patch? It avoids link errors while resolving abseil symbols with gcc
+-set_target_properties(check_epollexclusive PROPERTIES LINKER_LANGUAGE CXX)
```

Comment:
Why? And honestly for maintenance purpose, I don't recommend this strategy (a patch to our own CMakeLists wrapper).

Reply:
I thought about that, but figured this would be the less intrusive option. Easy enough change though.

Reply:
The idea is to maintain as less patches as possible for new versions.